app/views/2017/shared/_header.html.erb

```diff
@@ -15,8 +15,15 @@
       <li class="nav-link"><%= link_to t("header.search"),  "/search" %></li>
       <li class="nav-link"><%= link_to t("header.about"),   "/about" %></li>
     </ul>
+    <% unless I18n.locale == I18n.default_locale %>
```

Comment:
We could probably benefit from a blank line separating this thought and the previous group of nav links.

Reply:
Fixed.